### PR TITLE
Dashboard wiki page

### PR DESCRIPTION
**Pull request by andpopov**
@olenagerasimova Please review

### CODE REVIEW

File: .wiki/Dashboard.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Artipie dashboard
+
+The Artipie provides [front-end](github.com/artipie/front) web application to manage repositories using a web-browser.<br/>
+The Artipie front-end provides a convenient dashboard with UI pages for managing repositories.<br/>
+Artipie front-end is distributed as [Docker image](hub.docker.com/r/artipie/front) and as fat jar.
+The jar file can be downloaded on GitHub [releases page](https://github.com/artipie/front/releases).
+
+The Artipie front-end is independent part of Artipie project that interacts with Artipie server by using REST-API services provided by Artipie server.
```
Comment (olenagerasimova) [inline .wiki/Dashboard.md:8]: I'd suggest to add link to REST-API description in wiki

File: .wiki/Dashboard.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The Artipie dashboard provides following abilities:
```
Comment (olenagerasimova) [inline .wiki/Dashboard.md:10]: abilities -> functionality

File: .wiki/Dashboard.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+* Repository name
+* Repository type
+* Repository port where Artipie-server serves http requests to repository from according tools
+
+[[/images/dashboard/repository_list.jpg|Repository list]]
+
+The Artipie user can filter repositories by a repository name:
+[[/images/dashboard/repository_list_filtered.jpg|Filtered repository list]]
+
+## Create repository
```
Comment (olenagerasimova) [inline .wiki/Dashboard.md:41]: I'd suggest to add link to [this](https://github.com/artipie/artipie/wiki/Configuration-Repository) wiki page in repository subsection and write smth like "Follow this link to know all the technical details about supported repositories and settings".

File: .wiki/Dashboard.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Additionally page provides information how a new repository can be used by tools according a repository type.<br/>
+
+To save a new repository configuration please click `Add repository` button.
+[[/images/dashboard/repository_create|Create repository]]
+
+## Edit repository
+To edit existing repository please click on `repository name` on the `Repository list` page.
+
+The `Edit repository` page allows the Artipie user to change `configuration` of repository.<br/>
+Page provides information how the existing repository can be used by tools according a repository type.<br/>
```
Comment (olenagerasimova) [inline .wiki/Dashboard.md:59]: according  a repository -> according TO repository

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@andpopov thanks, check my suggestions please

File: .wiki/Dashboard.md (hunk 4 above)
Comment (andpopov) [reply .wiki/Dashboard.md]: @olenagerasimova thanks for important comments.
I have added fixes, please review

**Review by olenagerasimova — APPROVED** [review]
@andpopov thanks, approving

**Pull request merged**